### ISSUE TITLE
Testimonial Block: Alt Text for image does not adopt WordPress Settings

### ISSUE THREAD

**Issue by 00travelgirl00** [studiopress/atomic-blocks#380]
**Describe the bug**
If you insert a image in the Testimonial Block, the Alt Text from the WordPress Media Library does not appear. Instead the default alt Text "Avatar" appears. 

Screen Reader Users missing out the information of a more detailed and relevant information that are provided via alt-text in this way. 

Also Google prefers a more unique description, than 10 times "Avatar" in case you have 10 testimonials. 

This Problem does not apply to the "Profile Block" 

**To Reproduce**
1. Go to 'Media Library' and add Alt Text to image
2. Add "Testimonial" Block to Post and add the same image 
3. View Post in Frontend and check via Developer Tools the Alt Text
4. Alt-Text is "Avatar" and not the defined Alt-Text from Media Library 

**Expected behavior**
Alt Text should be the alt text from media library, not default "Avatar".
See "Profile" Block as an positive example. 

**Screenshots**
<img width="1167" alt="Bildschirmfoto 2020-06-08 um 10 26 54" src="https://user-images.githubusercontent.com/21318377/84011880-06e38d80-a977-11ea-9703-3fc6f2ede4c7.png">

**Version**
- Atomic Blocks 2.8.3
- WordPress 5.4.1 

**Device and browser information**
 - OS
 - Browser: Chrome
